### DISCUSSION TITLE
KEDA and DataDog compatibility

### DISCUSSION THREAD

**Discussion by ncolhoun-r7** [kedacore/keda#3469] [Q&A / Need Help] [answered]
Is there a workaround for the compatibility between KEDA and DataDog cluster agent? I seen this https://github.com/kedacore/keda/issues/470 issue was raised back in November 2019 but just wondered if there had been any progress made on running the two applications within the same cluster?

`helm install keda kedacore/keda --namespace keda
Error: rendered manifests contain a resource that already exists. Unable to continue with install: APIService "v1beta1.external.metrics.k8s.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-name" must equal "keda": current value is "datadog"; annotation validation error: key "meta.helm.sh/release-namespace" must equal "keda": current value is "kube-system"`

**K8s cluster version**
`WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.3", GitCommit:"aef86a93758dc3cb2c658dd9657ab4ad4afc21cb", GitTreeState:"clean", BuildDate:"2022-07-13T14:21:56Z", GoVersion:"go1.18.4", Compiler:"gc", Platform:"darwin/amd64"}
Kustomize Version: v4.5.4
Server Version: version.Info{Major:"1", Minor:"21+", GitVersion:"v1.21.12-eks-a64ea69", GitCommit:"d4336843ba36120e9ed1491fddff5f2fec33eb77", GitTreeState:"clean", BuildDate:"2022-05-12T18:29:27Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
WARNING: version difference between client (1.24) and server (1.21) exceeds the supported minor version skew of +/-1`

`helm version
version.BuildInfo{Version:"v3.6.3", GitCommit:"d506314abfb5d21419df8c7e7e68012379db2354", GitTreeState:"clean", GoVersion:"go1.16.5"}`

**Comment by tomkerkhove** [comment, marked as answer]
Hi, unfortunately the same limitation in Kubernetes upstream still applies and https://github.com/kubernetes-sigs/custom-metrics-apiserver/issues/70 is waiting for contributions.
